cmd/monaco/v2/delete/delete.go

```diff
@@ -59,31 +60,53 @@ func Delete(fs afero.Fs, deploymentManifestPath string, deletePath string) error
 		return errors.New("error while loading environments")
 	}
 
-	entriesToDelete, errors := configDelete.LoadEntriesToDelete(fs, getApiNames(apis), deleteFileWorkingDir, deleteFile)
+	entriesToDelete, errs := configDelete.LoadEntriesToDelete(fs, getApiNames(apis), deleteFileWorkingDir, deleteFile)
 
-	if errors != nil {
-		return fmt.Errorf("LoadEntriesToDelte throw an error: `%s`", errors)
+	if errs != nil {
+		return fmt.Errorf("LoadEntriesToDelte throw an error: `%s`", errs)
 	}
 
-	environments := manifest.Environments
+	environments := manifest.GetEnvironmentsAsSlice()
+
+	if len(envs) > 0 {
+		filtered, errs := environment.FilterEnvironmentsByName(environments, envs)
+
+		if errs != nil {
+			util.PrintErrors(errs)
+			return errors.New("error while loading environments")
+		}
+
+		environments = filtered
+	}
+
+	result := deleteConfig(environments, apis, entriesToDelete)
```

Comment:
```suggestion
	result := deleteConfigs(environments, apis, entriesToDelete)
```
nitpick: deletes more than one config